clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/deploy/converters/DeployCloudFoundryServerGroupAtomicOperationConverter.java

```diff
@@ -85,7 +85,7 @@ public DeployCloudFoundryServerGroupDescription convertDescription(Map input) {
     converted.setSpace(findSpace(converted.getRegion(), converted.getClient())
       .orElseThrow(() -> new IllegalArgumentException("Unable to find space '" + converted.getRegion() + "'.")));
 
-    Map artifactSource = (Map) input.get("artifactSource");
+    Map artifactSource = (Map) input.get("artifact");
```

Comment:
Can you elaborate on why the property name changed?

Reply:
The mapping from `artifact` to `artifactSource` happens in Orca. For the create server group operation we have control over that so it works fine, when sending the clone operation we don't have an easy extension point in Orca so we considered it easier to adjust this than to try and implement a custom operation for clone in Orca.

Reply:
Thanks @jmelchio!